warehouse/accounts/views.py

```diff
@@ -74,6 +76,17 @@ def failed_logins(exc, request):
     return resp
 
 
+@view_config(context=TooManyEmailsAdded, has_translations=True)
+def unverified_emails(exc, request):
+    return HTTPTooManyRequests(
+        _(
```

Comment:
I have _no_ idea why but it seems that translations don't work in this (or the other!) of these view configs. :/

https://sentry.io/share/issue/5a1bc4d04ca44ab1ad8043fd1dd0b28a/

What's even more frustrating is it _does_ work locally.